src/components/PlaylistPage/MobileComponents.js

```diff
@@ -1,7 +1,7 @@
 import React, {PropTypes} from 'react';
 import {LessonFilterContainer} from '../Filter/LessonFilter';
 import LevelNavigation from './LevelNavigation';
-import MobileButtonsRow from './MobileButtonsRow';
+import {MobileButtonsRowContainer} from './MobileButtonsRow.js';
```

Comment:
Ah, se der ja. Jeg gjorde noen endringer i MobileButtonsRow og MobileComponents tidligere (commit: Remove changes to mobilebuttonsrow because PR #239 fixes this), men fjernet de endringene, så må ha glemt å endre MobileButtonsRowContainer tilbake.